inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/doris/DorisSinkOperator.java

```diff
@@ -81,7 +82,10 @@ public StreamSink getFromEntity(@NotNull StreamSinkEntity entity) {
         }
 
         DorisSinkDTO dto = DorisSinkDTO.getFromJson(entity.getExtParams());
-        Preconditions.checkNotEmpty(dto.getFeNodes(), "doris fe nodes is empty");
+        if (dto.getFeNodes() == null) {
```

Comment:
Suggest to use` StringUtils.isNotBlank`.

Reply:
done